packages/ts/src/components/graph/modules/panel/helper.ts

```diff
@@ -26,26 +26,37 @@ export const DEFAULT_LABEL_MARGIN = 16
 export const OUTLINE_SELECTION_PADDING = 5
 export const DEFAULT_SIDE_LABEL_SIZE = 25
 
+export function getPanelPadding (padding: number | Spacing | undefined): Spacing {
+  const isPaddingAnObject = isPlainObject(padding)
+  return {
+    left: (isPaddingAnObject ? (padding as Spacing).left : (padding as number)) ?? DEFAULT_PADDING,
+    right: (isPaddingAnObject ? (padding as Spacing).right : (padding as number)) ?? DEFAULT_PADDING,
+    top: (isPaddingAnObject ? (padding as Spacing).top : (padding as number)) ?? DEFAULT_PADDING,
+    bottom: (isPaddingAnObject ? (padding as Spacing).bottom : (padding as number)) ?? DEFAULT_PADDING,
+  }
+}
+
+export function initPanels (panelsConfig: GraphPanelConfig[]): GraphPanel[] {
+  const panels = panelsConfig.map(p => ({
```

Comment:
We need to change it to `panelsConfig?.map` or set a default argument here because right now all of the graphs are broken when `panels` is not provided to the config.

<img width="447" alt="Screen Shot 2023-03-17 at 12 21 50 PM" src="https://user-images.githubusercontent.com/52078477/226000057-bf9c24a6-6706-4e70-8268-e081be1ed7df.png">

Reply:
I can't believe I missed that, thanks for pointing out! Fixed